src/components/navigation/nav.css

```diff
@@ -1,22 +1,30 @@
 header {
   margin-top: 0;
+  margin-bottom: 2rem;
   width: 100vw;
   height: 5rem;
   display: flex;
   align-items: center;
   justify-content: space-around;
-  background-color: #000;
-  color: #fff;
+  background-color: #fff;
+  border-bottom: 1px solid rgb(205, 205, 205);
   font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
 }
 
-button {
-  margin: 0 1rem 0 1rem;
-  background-color: transparent;
+.h1 {
+  color: #0290ff;
+  font-size: 2.5rem;
+  font-weight: 900;
+}
```

Comment:
- [ ] Please 🙏🏽 use the styles assigned to the title in the original design to match with Zeplin design 👇🏽 

![image](https://github.com/Booze33/book-store/assets/58642949/a9f98ab4-e7b2-49d2-bebd-d2c31c0fa8e5)